exporter/views.py

```diff
@@ -67,14 +69,40 @@ def do_bagit_export(assets, export_base_dir, export_filename_base):
 
         asset_id = asset_id.replace(":", "/")
         asset_path, asset_filename = os.path.split(asset_id)
+        item_path, item_filename = os.path.split(asset_path)
 
-        dest_path = os.path.join(export_base_dir, asset_path)
-        os.makedirs(dest_path, exist_ok=True)
+        asset_dest_path = os.path.join(export_base_dir, asset_path)
+        os.makedirs(asset_dest_path, exist_ok=True)
 
         # Build transcription output text file
-        text_output_path = os.path.join(dest_path, "%s.txt" % asset_filename)
-        with open(text_output_path, "w") as f:
+        asset_text_output_path = os.path.join(
+            asset_dest_path, "%s.txt" % asset_filename
+        )
+        with open(asset_text_output_path, "w") as f:
+            f.write(asset.latest_transcription or "")
+
+        item_text_output_path = os.path.join(asset_dest_path, "%s.txt" % item_filename)
+        with open(item_text_output_path, "a") as f:
             f.write(asset.latest_transcription or "")
+            f.write("\n\n")
+
+    # Add attributions to the end of all files found under asset_dest_path
+    if hasattr(settings, "ATTRIBUTION_TEXT"):
+        for dirpath, dirnames, filenames in os.walk(export_base_dir, topdown=False):
+            for each_text_file in filenames:
+                this_text_file = os.path.join(dirpath, each_text_file)
+                with open(this_text_file, "a") as f:
+                    f.write("\n\n")
+                    f.write(settings.ATTRIBUTION_TEXT)
+
+    # Make PDFs of all pages
+    for dirpath, dirnames, filenames in os.walk(export_base_dir, topdown=False):
+        for each_text_file in filenames:
+            this_pdf_file = each_text_file.replace(".txt", ".pdf")
+            pdfkit.from_file(
```

Comment:
Since the attribution text isn't part of the content, should we be using the `.txt` files or only including the actual transcription text and having pdfkit store it as document-level metadata or a footer?

Reply:
I think this is what we'll want to do eventually when we can implement the PDF creation bit as a separate service.